### ISSUE TITLE
Missing label for `textarea` input

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-modules/CSSEditor#11]
The `#css` `textarea` input is missing a label; the corresponding label element is assigned to the wrong ID (`custom-css`).
